prac_06/car.py

```diff
@@ -0,0 +1,31 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
```

Comment:
I think the multi line comment needs starting quotes and finishing quotes on a new line, What's your thoughts?